runtime/vm/jvminit.c

```diff
@@ -1878,12 +1878,13 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 					vm->vmRuntimeStateListener.idleTuningFlags &= ~(UDATA)J9_IDLE_TUNING_GC_ON_IDLE;
 				}
 				/*
-				 * CompactOnIdle is enabled only if:
+				 * CompactOnIdle is enabled only if -XX:+IdleTuningGcOnIdle is set and:
```

Comment:
This comment should be rewritten when the code block is moved as it indicates `-XX:+IdleTuningGcOnIdle` both here and on the next line

Reply:
@DanHeidinga  I'm sorry, I didn't quite get you. Do you mean changing the comment to 
```
 CompactOnIdle is enabled only if: 
1. -XX:+IdleTuningCompactOnIdle is set, or
2. running in container 
```

Reply:
(My original comment was wrong - I had misread it to specify `-XX:+IdleTuningGcOnIdle` twice - sorry)

I'd like the comment to explain the rationale for how /  why this option gets set.  It should include the explanation that CompactOnIdle can only be enabled if `-XX:+IdleTuningGcOnIdle` is already set.

It should indicate the difference from the `-XX:+IdleTuningGcOnIdle` case in that compact isn't enabled by -Xtune:virtualized.

Reply:
Updated the comment